docs/modules/ROOT/pages/component-loki.adoc

```diff
@@ -0,0 +1,21 @@
+= Loki
+
+[abstract]
+Loki can be used to process and store Couchbase logs.
+
+== Overview
+
+https://grafana.com/oss/loki/[Loki^] is the https://prometheus.io/[Prometheus^]-inspired open source, multi-tenant log aggregation system. Loki is used to process and store collected Couchbase logs.
+
+Couchbase cluster generates logs of various operations which is important for monitoring and investigation purposes. These logs need to be parsed in order to search and aggregate them based on keys. Fluent bit forwards the couchbase logs to loki which enables processing and storage of logs.
```

Comment:
Again, split up the sentences.

Something needs to forward the logs, we recommend Fluent Bit and provide a method to deploy it.

Loki could also be used to store other logs by a customer potentially too. They may have their own instance already, or be using Grafana Cloud as well.

Reply:
>Loki could also be used to store other logs by a customer potentially too. They may have their own instance already, or be using Grafana Cloud as well.

Do we need to mention this? We have already answered the question, "Why we need Loki and how does that fit in with Cloud native approach"

Reply:
Addressed the rewording